wasmtime/_error.py

```diff
@@ -1,12 +1,16 @@
 from ctypes import byref
+import typing
+
+if typing.TYPE_CHECKING:
+    from ._ffi import P_wasmtime_error_t
```

Comment:
How come this is conditional?

Reply:
I'm avoiding importing anything into the namespace just for type-checking.